ibm/service/iamidentity/resource_ibm_iam_account_settings_test.go

```diff
@@ -126,9 +128,15 @@ func testAccCheckIbmIamAccountSettingsUpdateConfig() string {
 			restrict_create_platform_apikey = "%s"
 			if_match = "%s"
 			mfa = "%s"
+			user_mfa {
+				iam_id = "iam_id"
```

Comment:
can we add a test case to add multiple users and update operation to remove or add users

Reply:
I have added two tests, one is to test with an empty list of user_mfa, and another one with 2 users.